system/infra-monitoring/vendor/vasa-exporter/alerts/vasa.alerts

```diff
@@ -0,0 +1,32 @@
+groups:
+- name: vasa.alerts
+  rules:
+  - alert: VasaFilesystemUsageHigh
+    expr: vasa_disk_utilization_percentage > 74
+    for: 5m
+    labels:
+      severity: warning
+      tier: vmware
+      service: vasa
+      context: vasa
+      meta: "VASA filesystem in {{ $labels.server_name }} has high utilization for 5 min."
+      playbook: 'docs/devops/alert/vmware/
+      dashboard: vasa-provider?var-az={{ $labels.server_name }}
+    annotations:
+      description: "VASA filesystem in {{ $labels.server_name }} has 70% or more utilization for 5 min."
+      summary: "VASA filesystem in {{ $labels.server_name }} has high utilization for 5 min."
+
+  - alert: VasaFilesystemUsageCritical
+    expr: vasa_disk_utilization_percentage > 89
+    for: 5m
+    labels:
+      severity: critical
+      tier: vmware
+      service: vasa
+      context: vasa
+      meta: "VASA filesystem in {{ $labels.server_name }} has a critical utilization for 5 min."
+      playbook: docs/devops/alert/vcenter/#vasa_filesystem_usage_high
+      dashboard: vasa-provider?var-az={{ $labels.server_name }}
+    annotations:
+      description: "VASA filesystem in {{ $labels.server_name }} has 90% or more utilization for 5 min."
+      summary: "VASA filesystem in {{ $labels.server_name }} has critical utilization for 5 min."
```

Comment:
it seems that an empty line is needed after the block

Reply:
why?

Reply:
https://github.com/sapcc/helm-charts/blob/master/system/infra-monitoring/vendor/vcenter-exporters/alerts/vcenter.alerts

There is also no empty line at the end.

Reply:
the newline is not the problem, it is a typo ;)